util/topgen/validate.py

```diff
@@ -369,6 +369,7 @@
 }
 reset_request_optional = {
     'width': ['d', 'TODO'],
+    'reset_enabled': ['pb', 'whether the reset request is enabled'],
```

Comment:
```suggestion
    'enabled_after_reset': ['pb', 'whether the reset is enabled after a reset (put differently, whether the reset value of the reset enable is high)'],
```
Maybe make this attribute name slightly more verbose to clarify what it does?